CONTRIBUTING.md

```diff
@@ -1,5 +1,13 @@
 # Contributing
 
+## ariakit
+
+We're using [@ariakit-react](https://ariakit.org) to build many of the components in Pluto.
+Currently we're still using an outdated version of it (formerly named
+[reakit](https://reakit.io/docs/get-started/)). For some of the components you might to look at the
+old versions documentation until we upgraded to the new version (
```

Comment:
```suggestion
old version's documentation until we upgraded to the new version)
```

Reply:
Also
```suggestion
old versions documentation until we upgrade to the new version (
```
right?

Reply:
This specifically refers to the point in time after we upgraded. So the upgrade event should be in the past.